src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -687,7 +693,7 @@ class GrowableArrayCHeap : public GrowableArrayWithAllocator<E, GrowableArrayCHe
   }
 
 public:
-  GrowableArrayCHeap(int initial_max) :
+  GrowableArrayCHeap(int initial_max = 0) :
```

Comment:
Just noting that GrowableArray defaults to 2 and here we default to 0.

Reply:
Thanks for reviewing. For a CHeap allocater array I think an initial_max of 0 makes more sense, since the cost of doing an allocation is typically higher, and lazily allocating the array means we will avoid the allocation cost all together in cases where an array was created but elements were never added.